api/tests/admin/custom_views/support_view_test.py

```diff
@@ -131,6 +131,32 @@ def test_validation_view_validate_user_from_dms_data_staging(self, client):
         assert user.lastName == dms_content.last_name
         assert user.idPieceNumber == dms_content.id_piece_number
 
+    @override_features(BENEFICIARY_VALIDATION_AFTER_FRAUD_CHECKS=True)
+    @override_features(IS_HONOR_STATEMENT_MANDATORY_TO_ACTIVATE_BENEFICIARY=True)
+    def test_validation_view_validate_user_from_educonnect(self, client):
+        user = users_factories.UserFactory(dateOfBirth=datetime.utcnow() - relativedelta(years=15, months=2))
+        check = fraud_factories.BeneficiaryFraudCheckFactory(user=user, type=fraud_models.FraudCheckType.EDUCONNECT)
+        admin = users_factories.AdminFactory()
+        client.with_session_auth(admin.email)
+
+        response = client.post(
+            f"/pc/back-office/support_beneficiary/validate/beneficiary/{user.id}",
+            form={"user_id": user.id, "reason": "User is granted", "review": "OK"},
+        )
+        assert response.status_code == 302
+
+        review = fraud_models.BeneficiaryFraudReview.query.filter_by(user=user, author=admin).one()
+        assert review.review == fraud_models.FraudReviewStatus.OK
+        assert review.reason == "User is granted"
+        user = users_models.User.query.get(user.id)
+        assert user.has_underage_beneficiary_role is True
+        assert len(user.deposits) == 1
+
+        educonnect_content = fraud_models.EduconnectContent(**check.resultContent)
+        assert user.firstName == educonnect_content.first_name
+        assert user.lastName == educonnect_content.last_name
+        assert user.idPieceNumber == None
```

Comment:
(détail) `assert not user.idPieceNumber` 

ou `assert user.idPieceNumber is None` si on peut avoir `False` ou une autre valeur qui sera évaluée à `False` et qu'on veut s'assurer qu'il s'agit bien de `None`)

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.
https://www.python.org/dev/peps/pep-0008/#programming-recommendations